templates/base.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    {% block title %}
+        <title>Taxi Service</title>
+    {% endblock %}
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+    {% load static %}
+    <link rel="stylesheet" href="{% static 'css/styles.css' %}">
+</head>
+<body>
+    {% block sidebar %}
+    {% endblock %}
+
+    {% block content %}
+    {% endblock %}
+
+    {% block pagination %}
+    {% endblock %}
+</body>
+</html>
```

Comment:
please, add a blank line at the end of the file

Reply:
Of course, thanks